src/pybind/mgr/dashboard/controllers/rgw.py

```diff
@@ -231,9 +231,11 @@ def get_s3_bucket_name(bucket_name, tenant=None):
             bucket_name = '{}:{}'.format(tenant, bucket_name)
         return bucket_name
 
-    def list(self, stats=False, daemon_name=None):
-        # type: (bool, Optional[str]) -> List[Any]
-        query_params = '?stats' if str_to_bool(stats) else ''
+    def list(self, stats: bool = False, daemon_name: Optional[str] = None,
```

Comment:
Should this be versioned?

Reply:
I guess so. We agreed on skipping the versioning for the NFS endpoints so far since the existing implementation wasn't functional, but the RGW ones should.

Reply:
@epuertat @avanthakkar I know. Done, but bear in mind that now we need the PR that provides api minor version retrocompatibility otherwise this endopint will throw an error from now on.